### PR TITLE
(TK-350) Prep for 1.3.0 release

### PR DESCRIPTION
**Pull request by cprice404**
This commit updates all of the dependencies to the latest
versions of the various PL/TK components, to prepare for a
new 1.3.0 release of the template.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(defproject trapperkeeper/lein-template "1.1.0"
+(defproject trapperkeeper/lein-template "1.3.0-SNAPSHOT"
```
Comment (rlinehan) [inline project.clj]: why are we bumping from 1.1.0 to 1.3.0?

Comment (cprice404) [inline project.clj]: I arbitrarily chose that to try to make the Y match up with the related version of TK itself.  I'm not attached to that decision and would be happy to change this to whatever other number is preferred if anyone has a different opinion.

Comment (rlinehan) [inline project.clj]: Hm, okay. I guess that makes sense. I would maybe worry about what happens if we make changes to the template other than just bumping the tk version, what do we bump the version to then? But version numbers are terrible and I don't want to think about them too much so I'm fine with whatever.

**Comment by rlinehan** [comment]
👍

**Comment by fpringvaldsen** [comment]
:+1:

**Comment by rlinehan** [comment]
Seems like we've mostly versioned this according to TK version in the past, and since Preben's +1'd this I'll merge.

**Pull request merged**